phalcon/cache/backend/factory.zep

```diff
@@ -45,7 +45,7 @@ class Factory extends BaseFactory
 	/**
 	 * @param \Phalcon\Config|array config
 	 */
-	public static function load(var config) -> <BackendInterface>
+	public static function load(var config) -> object
```

Comment:
This one I don't think we can change it to the `BackendInterface` because it inherits from `FactoryInterface` which is used for other factory classes (say `Config`, `Paginator` etc.) If this is not `object` then we have interface problems in compilation